packages/queued-request-controller/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Bump `@metamask/json-rpc-engine` from `^7.1.0` to `^7.2.0` ([#1895](https://github.com/MetaMask/core/pull/1895))
```

Comment:
FYI: This will get added in https://github.com/MetaMask/core/pull/2004, so may want to wait a bit until that is merged (the DevEx team is in Eastern time).

Reply:
Removed in https://github.com/MetaMask/core/pull/2003/commits/6a5f2f3c626f5e45fa1864f1cdb817014afcfc8e!